app/views/users/_sidebar.html.erb

```diff
@@ -3,45 +3,18 @@
   <div class="side-bar">
     <div class="user-sidebar">
       <% cache "user-profile-sidebar-profile-details-#{@user.id}-#{@user.profile_updated_at}", expires_in: 10.days do %>
-        <% if @user.mostly_work_with.present? %>
-          <div class="widget">
-            <header>
-              <h4>skills/languages</h4>
-            </header>
-            <div class="widget-body">
-              <%= sanitized_sidebar @user.mostly_work_with %>
+        <% ProfileField.where(display_area: :left_sidebar).each do |pf| %>
```

Comment:
+1 to extracting this out of the view and making the controller responsible for setting this (and passing it along to the view)

Reply:
Normally, yes. 

For quick fixes that I don't expect to be around long, I often apply different rules: the fewer files I have to touch, the better. The logic (do we have a value and should we render something?) can not easily move into the controller and this draft PR was not the right place to come up with the proper abstraction. It was only meant to enable the Quidditch Forem to quickly finalize their profile setup and then be ripped out again ASAP. 

Now that we deployed the disabled feature flag for everyone, I have more time, so this will definitely move _somewhere_.